Extensions/DialogueTree/dialoguetools.js

```diff
@@ -279,9 +284,10 @@ gdjs.dialogueTree.getLineOptionsCount = function() {
  */
 gdjs.dialogueTree.confirmSelectOption = function() {
   if (
-    this.dialogueData.select &&
-    !this.selectedOptionUpdated &&
-    this.selectedOption !== -1
+    !this.dialogueIsRunning ||
```

Comment:
Same are you sure about your condition?

Reply:
yes, this prevents it from trying to read data when no dialogue is initiated

Reply:
oops, sorry, this is not right. Will amend now